### ISSUE TITLE
Registration Form Improvements

### ISSUE THREAD

**Issue by Kavakuo** [Kavakuo/LabHive#24]
We want to split up the registration form in multiple to parts, to structure it a little bit more. We need 3 roles of users in the platform
* Research Labs
* Diagnostic Labs
* Volunteers

Each of them needs a slightly different registration form. The idea is to split the registration form into common (every user has to enter an email and password) and user specific tasks.

An idea is visible in the [mockup](https://www.notion.so/Application-for-Solution-Enabler-ee461ab6367b4cf784de33b08827f61e#cdd1be3e99bb41a682c25563db599140).

So basically every step needs a button at the end that takes the user to the next step. The data is collected over the steps and send to backend with the last button click. I think it would be nice if the transition between the steps is animated ([Transitions between components](https://vuejs.org/v2/guide/transitions.html#Transitioning-Between-Components)).
